tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_download_feeds.yaml

```diff
@@ -156,6 +156,17 @@
     download_timeout: 120
     update_treshold_weeks: 3
 
+- name: 'ALAS-2022'
+  description: 'Amazon Linux provider'
+  configuration_parameters:
+    PROVIDER: 'alas'
+    OS: 'amazon-linux-2022'
+  metadata:
+    provider_name: 'Amazon Linux 2022'
+    provider_os: 'Amazon-Linux-2022'
+    download_timeout: 120
```

Comment:
I don't remember if I asked this in the previous PR, but Did you verify that the `download_timeout` value is correct?